packages/charts/backend/sirius-components-collaborative-charts/src/main/java/org/eclipse/sirius/components/collaborative/charts/HierarchyEventFlux.java

```diff
@@ -55,7 +55,9 @@ public void hierarchyRefreshed(IInput input, Hierarchy newHierarchy) {
     }
 
     public Flux<IPayload> getFlux(IInput input) {
-        var initialRefresh = Mono.fromCallable(() -> new HierarchyRefreshedEventPayload(input.id(), this.currentHierarchy));
+        var initialRefresh = Mono.fromCallable(() -> {
```

Comment:
Is there anything that should change here?

Reply:
My mistake. I had changed the formatting to put a breakpoint on the inside of the lambda and forgot to revert.